### ISSUE TITLE
Expose "getNode" Method on the Grid Client

### ISSUE THREAD

**Issue by Mahmoud-Emad** [threefoldtech/tfgrid-sdk-ts#481]
### Description

Currently, the "getNode" method is not available on the Grid Client, limiting its functionality and access to essential node information. This issue aims to address this limitation by implementing the "getNode" method within the Node class of the chain_client module and subsequently exposing it in the grid_client module.

### Implementation

- Implement the "getNode" method inside the Node class located in the `chain_client` module. This method will utilize the Polkadot API to fetch relevant node data. Additionally, ensure the data is wrapped in appropriate interfaces to enhance code readability and maintainability.

- Add the implemented "getNode" method to the grid_client module, making it accessible and available for registration within the client. This step will enable users of the Grid Client to retrieve node information conveniently.

**Comment by Mahmoud-Emad** [comment]
Should be fixed in #482 and #485

**Comment by mohamedamer453** [comment]
Verified on devnet.

The `get` function was successfully added to the `nodes` module and its' working.

I verified it locally by creating a simple script that calls the function and it worked without any issues.


```ts
import { getClient } from "./client_loader";
import { log } from "./utils";

async function main() {
  const client = await getClient();

  const res = await client.nodes.get({id: 11})
  log(res);


  await client.disconnect();
}

main();

```

![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/101194226/04a41329-ca69-4bdb-9bfe-dff3979ebe91)
